lib/apiservers/portlayer/swagger.json

```diff
@@ -2544,11 +2574,12 @@
 					}
 				},
 				"Store": {
+					"x-nullable": false,
 					"type": "string"
 				},
 				"Capacity": {
-					"type": "integer",
-					"format": "int64"
+					"x-nullable": false,
+					"type": "integer"
```

Comment:
The version of Swagger we used caused an int64 to get generated if we tag the parameter with format.  Latest version of Swagger no longer does this?